app/src/cli/doc.rs

```diff
@@ -0,0 +1,29 @@
+use docs::open;
+use docs::write_html;
+use driver::paths::DOCS_PATH;
+use std::path::PathBuf;
+
+#[derive(clap::Args)]
+pub struct Args {
+    #[clap(value_parser, value_name = "FILE")]
+    filepath: PathBuf,
+    #[clap(long, num_args = 0)]
+    open: bool,
+}
+
+pub fn exec(cmd: Args) -> miette::Result<()> {
+    let htmlpath = get_path(&cmd);
+    let filepath = &cmd.filepath;
+    write_html(filepath, &htmlpath);
+    if cmd.open {
+        open(&htmlpath);
+    }
+    Ok(())
+}
+
+fn get_path(cmd: &Args) -> PathBuf {
+    let path = format!("{}{}", DOCS_PATH, cmd.filepath.file_name().unwrap().to_string_lossy());
```

Comment:
Use [`Path::join`](https://doc.rust-lang.org/std/path/struct.Path.html#method.join) here instead of `format!`.